pages/purchase.tsx

```diff
@@ -40,10 +40,12 @@ export default function Home() {
         const response = await fetch(
           `https://api.tzkt.io/v1/accounts/${address}/balance`
         );
-        const data = await response.json();
-        setBalance(data / 1000000);
+        const MUTEZ = 1000000;
+        const MUTEZBalance = await response.json();
+        const FinalBalance = MUTEZBalance / MUTEZ;
+        setBalance(FinalBalance);
```

Comment:
Thanks Daniel for update, there are some further updates I'd like you to address. 

I want you to go https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx and verify whether your variable names satisfy the conventions.